### ISSUE TITLE
Cannot run actions twice in a single workflow

### ISSUE THREAD

**Issue by jozefizso** [Azure/trusted-signing-action#35]
Running the actions twice will results in failed workflow:

```
Write-Error: Failed to create file 'D:\a\_actions\azure\trusted-signing-action\v0.3.19\TrustedSigning\about_TrustedSigning.help.txt'
while expanding the archive file 'D:\a\_actions\azure\trusted-signing-action\v0.3.19\trusted-signing-module-0.3.18.zip'
contents as the file 'D:\a\_actions\azure\trusted-signing-action\v0.3.19\TrustedSigning\about_TrustedSigning.help.txt'
already exists. Use the -Force parameter if you want to overwrite the existing directory
'D:\a\_actions\azure\trusted-signing-action\v0.3.19\TrustedSigning\about_TrustedSigning.help.txt' contents when
expanding the archive file.
```

**Comment by jayden-macrae** [comment]
Have you tried:
```
uses: azure/trusted-signing-action@v0.3.20
```
To use the latest version published 4 days ago which was meant to address this?

**Comment by dlemstra** [comment]
This is a duplicate of #30 and it should be fixed in the latest release.

**Comment by japarson** [comment]
Hi @jozefizso, this issue should be fixed in v0.3.20. Please try it out and close this issue if you're no longer hitting this error. Thank you!
